docs/cli_command_reference.md

```diff
@@ -0,0 +1,421 @@
+# NEAR AI CLI Command Reference
+
+This document provides a detailed reference of all available CLI commands in NEAR AI, including their syntax, options, and example usage.
+
+## Table of Contents
+
+1. [Registry Commands](#registry-commands)
+2. [Config Commands](#config-commands)
+3. [Benchmark Commands](#benchmark-commands)
+4. [Evaluation Commands](#evaluation-commands)
+5. [Agent Commands](#agent-commands)
+6. [Hub Commands](#hub-commands)
+7. [Login Commands](#login-commands)
+8. [Logout Command](#logout-command)
+9. [Permission Commands](#permission-commands)
+10. [Miscellaneous Commands](#miscellaneous-commands)
+
+## Registry Commands
+
+### `nearai registry info`
+
+Show information about an item in the registry.
+
+Syntax:
+```
+nearai registry info <entry>
+```
+
+Example:
+```
+nearai registry info namespace/agent_name/1.0.0
+```
+
+### `nearai registry metadata-template`
+
+Create a metadata template for a registry item.
+
+Syntax:
+```
+nearai registry metadata-template [--local_path=<path>] [--category=<category>] [--description=<description>]
+```
+
+Example:
+```
+nearai registry metadata-template --local_path=./my_agent --category=agent --description="My new agent"
+```
+
+### `nearai registry list`
+
+List available items in the registry.
+
+Syntax:
+```
+nearai registry list [--namespace=<namespace>] [--category=<category>] [--tags=<tags>] [--total=<total>] [--offset=<offset>] [--show_all] [--show_latest_version] [--star=<star>]
+```
+
+Example:
+```
+nearai registry list --category=agent --tags=nlp,chatbot --total=10
+```
+
+### `nearai registry update`
+
+Update metadata of a registry item.
```

Comment:
We should add more information about some of these commands and ideally test all of them out so we know how they work. This one, for example, only updates the _remote_ metadata.json file from information that is found locally.